tests/callbacks_test.py

```diff
@@ -0,0 +1,23 @@
+import pytorch_lightning as pl
+from torch import nn
+
+from bolts.callbacks import PostHocEval
+from bolts.fair.models import ErmBaseline
+from tests.fair.model_test import DummyDataModule, Encoder  # type: ignore[import]
```

Comment:
why can't mypy import this?

Reply:
because `tests` is a dir not a module

Reply:
i read somewhere that this is a preferred way of structuring tests, but now I can't find it, and can only find sources that say _do_ include `__init__.py`

Reply:
I added __init__.py files... still can't find them -_-

Reply:
ah, was missing one